src/structures/Channel.js

```diff
@@ -97,6 +97,14 @@ class Channel extends Base {
     return this.client.channels.fetch(this.id, true, force);
   }
 
+  /**
+   * Whether this channel is text-based
```

Comment:
```suggestion
   * Whether this channel is text-based.
```

Reply:
There are descriptions both with and without the period on the `Channel` class, but after having a look at them I think without it would be the better approach

Reply:
All methods are to have periods per all other methods in our documentation.